lib/segment/src/index/vector_index_base.rs

```diff
@@ -51,7 +51,11 @@ pub trait VectorIndex {
     fn indexed_vector_count(&self) -> usize;
 
     /// Update index for a single vector
-    fn update_vector(&mut self, id: PointOffsetType, vector: VectorRef) -> OperationResult<()>;
+    fn update_vector(
```

Comment:
please add a comment regarding the meaning of the optional vector :)

Reply:
Done